apps/sms/test/marionette/composer_test.js

```diff
@@ -322,5 +322,13 @@ marionette('Messages Composer', function() {
       assert.deepEqual(contact.name, newMessage.recipients);
       assert.equal(contact.tel[0].value, newMessage.recipientsPhoneNumbers);
     });
+
+    test('should accept semicolon to completes the entered recipient',
+    function() {
+      var separator = ';';
+      newMessage.addNewRecipient(123, separator);
```

Comment:
Could you please extend this test a bit, and add one more recipient so that we can be sure that recipient editor is created and ready for input? Smth like:
```
var separator = ';';
newMessage.addNewRecipient(123, separator);
assert.lengthOf(newMessage.recipients, 1);
assert.notInclude(newMessage.recipients[0], separator);

newMessage.addNewRecipient(345, separator);
assert.lengthOf(newMessage.recipients, 2);
assert.notInclude(newMessage.recipients[1], separator);
```